datafeeder/src/docker/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM openjdk:8-jre
+FROM bellsoft/liberica-openjdk-alpine-musl:11
```

Comment:
I'm a bit concerned by this move for production use (image is 2 months old)
cc @pmauduit

Reply:
Motivated by trying a thousand things that could make the bug show in docker and not in a local app run. We can get rid of it.

Reply:
@pmauduit would you be so kind of reverting the Dockerfile commit for me? I'm unavailable until afternoon

Reply:
sure